src/RangeSlider.elm

```diff
@@ -27,15 +27,27 @@ onChange msg input =
     Html.Events.on "change" (Json.Decode.map msg input)
 
 
-sliderInputView : CommonAttributes -> ValueAttributes msg -> Json.Decode.Decoder Float -> Html msg
-sliderInputView commonAttributes valueAttributes input =
+sliderInputView : CommonAttributes -> ValueAttributes msg -> Json.Decode.Decoder Float -> Maybe String -> Html msg
+sliderInputView commonAttributes valueAttributes input extraClass =
+    let
+        baseClass =
+            "input-range"
+
+        class =
+            case extraClass of
+                Just s ->
+                    baseClass ++ " " ++ s
+
+                Nothing ->
+                    baseClass
+    in
     Html.input
         [ Html.Attributes.type_ "range"
         , Html.Attributes.min <| String.fromFloat commonAttributes.min
         , Html.Attributes.max <| String.fromFloat commonAttributes.max
         , Html.Attributes.step <| String.fromFloat commonAttributes.step
         , Html.Attributes.value <| String.fromFloat valueAttributes.value
-        , Html.Attributes.class "input-range"
+        , Html.Attributes.class class
```

Comment:
Since you can have as many `.class` as you like
https://package.elm-lang.org/packages/elm/html/latest/Html-Attributes#class

It should be possible to do something like
```elm
, Html.Attributes.class "input-range"
, Html.Attributes.class (Maybe.withDefault "" extraClass)
```

Which would get rid of the `let`